WordPress/Classes/ViewRelated/Me/Help & Support/Activity Logs/ActivityLogViewController.m

```diff
@@ -141,16 +141,49 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                                                                                                        forDateString:[self.dateFormatter stringFromDate:logFileInfo.creationDate]];
         [self.navigationController pushViewController:detailViewController animated:YES];
     } else {
-        for (DDLogFileInfo *logFileInfo in self.logFiles) {
-            if (logFileInfo.isArchived) {
-                [[NSFileManager defaultManager] removeItemAtPath:logFileInfo.filePath error:nil];
-            }
-        }
-
-        DDLogWarn(@"All archived log files erased.");
-
-        [self loadLogFiles];
-        [self.tableView reloadData];
+        //Delete old activity logs
+        NSString *titleText = NSLocalizedString(@"Delete", comment: @"Title of the trash confirmation alert.");
+        NSString *messageText = NSLocalizedString(@"Clear all old activity logs?", comment: @"Message of the trash confirmation alert.");
+        NSString *trashButtonTitle = NSLocalizedString(@"Yes", comment: @"Label for a button that clears all old activity logs");
+        NSString *cancelButtonTitle = NSLocalizedString(@"No", comment: @"Label for a cancel button");
+        
+        UIAlertController * alert = [UIAlertController
```

Comment:
Our preferred code style is to have the asterisk "cuddle" the variable name (like you see above).  So this would be `UIAlertController *alert`.

Reply:
Sure, adjusted the position of the asterisk in https://github.com/wordpress-mobile/WordPress-iOS/pull/13799/commits/6e008a5840adc2acb87b968ef2714d877e389832